ui/src/authorizations/apis/index.ts

```diff
@@ -20,7 +21,7 @@ export const createAuthorization = async (
 
 export const getAuth0Config = async (): Promise<Auth0Config> => {
   try {
-    const response = await fetch('/api/v2private/oauth/clientConfig')
+    const response = await fetch(`${getAPIBasepath()}/api/v2private/oauth/clientConfig`)
```

Comment:
AJAX injects this by default, fetch does not, shouldn't have a v2private in the first place, needed to add this to get my dev env to work

Reply:
i think the longer term solution to this is being addressed as part of the larger "restructure the ui in 3 places problem" task coming this quarter, this was just a weird anomaly.

Reply:
that's really strange since this was actually returning the correct data when toggled on acceptance. Would you mind elaborating about the defaults a bit?

Reply:
sure! all of our cloud environments run with a default api prefix of `/`, but for oss, people need the ability to run the frontend under an arbitrary prefix. i run it under `/influx` or `/metrics` for my personal projects, as my introspection tooling isn't the application itself, but rides shotgun.

here's where it happens for ajax requests:
https://github.com/influxdata/influxdb/blob/master/ui/src/utils/ajax.ts#L6

and here's where it happens for the swagger defined endpoints:
https://github.com/influxdata/influxdb/blob/master/ui/src/client/index.ts#L10

i think it's an edgecase to add api requests outside of those two avenues, so we dont have to auto fill fetch or anything, and we dont do it perfectly everywhere (im looking at you webpack-dev-server), i'm just stoked to share the concept of prefixing your app so that you can embed frontends into larger url hierarchies without forcing their code to be linked.